exp-player/addon/components/exp-exit-survey.js

```diff
@@ -95,6 +97,20 @@ export default ExpFrameBaseComponent.extend({
                     type: 'jsonschema',
                     description: 'JSON-schema defining second form.',
                     default: emailOptOutSchema
+                },
+                idealSessionsCompleted: {
```

Comment:
From a reusability standpoint, having these default values live in the component would seem to lock us to the assumption that every use of this component is a longitudinal study.

It might be better to use default values that trigger no behavior at all, rather than hardcoding specific default numbers.